modules/host.go

```diff
@@ -92,6 +92,12 @@ type (
 		ObligationStatus    uint64 `json:"obligationstatus"`
 	}
 
+	// HostWorkingState reports the working state of a host.
+	HostWorkingState string
+
+	// HostConnectabilityState reports the connectability state of a host.
+	HostConnectabilityState string
```

Comment:
nvm I see that you did define constants for each potential value. Still they need to be enumerated in this docstring though.